src/main/java/com/cloudbees/jenkins/plugins/sshagent/jna/JNRRemoteAgentStarter.java

```diff
@@ -24,15 +24,23 @@
 
 package com.cloudbees.jenkins.plugins.sshagent.jna;
 
+import jenkins.security.MasterToSlaveCallable;
+
 import com.cloudbees.jenkins.plugins.sshagent.RemoteAgent;
+
 import hudson.model.TaskListener;
-import hudson.remoting.Callable;
 import hudson.remoting.Channel;
 
 /**
  * Callable to start the remote agent.
  */
-public class JNRRemoteAgentStarter implements Callable<RemoteAgent, Throwable> {
+public class JNRRemoteAgentStarter extends MasterToSlaveCallable<RemoteAgent, Throwable> {
+
+    /**
+     * 
+     */
+    private static final long serialVersionUID = -8381290066841681157L;
```

Comment:
```
$ serialver -classpath /Users/stephenc/.m2/repository/org/jenkins-ci/plugins/ssh-agent/1.7/ssh-agent-1.7.jar:/Users/stephenc/.m2/repository/org/jenkins-ci/main/remoting/2.33/remoting-2.33.jar:/Users/stephenc/.m2/repository/org/jenkins-ci/main/jenkins-core/1.532.2/jenkins-core-1.532.2.jar  com.cloudbees.jenkins.plugins.sshagent.jna.JNRRemoteAgentStarter
com.cloudbees.jenkins.plugins.sshagent.jna.JNRRemoteAgentStarter:    private static final long serialVersionUID = 5020446864184061252L;
```

`private static final long serialVersionUID = 5020446864184061252L;` is what we want